### PR TITLE
Add Python Numpy and Pandas Cheat Sheet

### PR DESCRIPTION
**Pull request by g-paras**
Added 
-  `Pandas` Cheat Sheet for Data Science in Python
-  `Numpy` Cheat Sheet for Data Analysis in Python

**Source: _Data Camp_**

### CODE REVIEW

File: cheatSheets.md [hunk 1]
```diff
@@ -47,6 +47,11 @@
 - [Flutter Cheat Sheet](https://howtodothisinflutter.com/): Basic flutter commands and sample code
 
 - [HTML Cheat Sheet](https://html.com/wp-content/uploads/html-cheat-sheet.pdf): This clear, easy-to-access PDF thoroughly explains HTML elements, with understandable explanations and examples.
+-
```
Comment (AvidCoder101) [inline cheatSheets.md:50]: Please remove this unnecessary bullet point @g-paras

**Review by AvidCoder101 — CHANGES_REQUESTED** [review]

File: cheatSheets.md (hunk 1 above)
Comment (g-paras) [reply cheatSheets.md]: Done 👍🏻

**Review by AvidCoder101 — APPROVED** [review]
LGTM, @arjunkharbanda I think that this PR can be merged

**Comment by arjunkharbanda** [comment]
@AvidCoder101  Thanks for reviewing it. You can  merge it by yourself you don't have to wait for me if it looks good to you it's fine by  me just make sure it passes all our check and no conflicts are there

**Comment by AvidCoder101** [comment]
@arjunkharbanda I can't seem to merge because I am not an admin, so that's why I ask one of the owners of the organization to merge.

**Comment by arjunkharbanda** [comment]
@AvidCoder101  I didn't know that. I can add you as maintainer of this repository if you want or you can join @zero-to-mastery/gitgang  but you have to ask  @MattCSmith  about that.

**Comment by AvidCoder101** [comment]
@arjunkharbanda I would appreciate it if you could add me as a maintainer of this repo. Thank you 🙂

**Comment by arjunkharbanda** [comment]
@AvidCoder101 sorry for the delay got caught up with something, Anyways  now you  are officially maintainer of this repository I hope you continue your support and maintain this repository

**Comment by AvidCoder101** [comment]
Thank you @arjunkharbanda

**Pull request merged**